doc/_scripts/software_maturity/software_maturity_features.yaml

```diff
@@ -132,6 +132,12 @@ features:
         - SHIELD_NRF7002EB
         - BOARD_NRF7002DK_NRF5340_CPUAPP
         - BOARD_NRF7002DK_NRF5340_CPUAPP_NRF7001
+  fast_pair:
+    Initial Pairing: BT_FAST_PAIR
+    Subsequent pairing: BT_FAST_PAIR && EXPERIMENTAL
```

Comment:
Can we combine initial and subsequent pairing (it's controlled by the same Kconfig symbol)

Reply:
We want to declare different maturity level for both and as you indicated we will have a dedicated Kconfig for enabling support for subsequent pairing in the future

Reply:
I am worried that we may need a separate Kconfig for subequent pairing in that case. Current solution may work by an accident (we are looking for configurations with both BT_FAST_PAIR && EXPERIMENTAL enabled). Similar for battery notification.